Avro serdes support (#407)
quixio/quix-streams: docs/advanced/serialization.md

input_topic = app.topic('input', value_deserializer=JSONDeserializer(schema=MY_SCHEMA))
output_topic = app.topic('output', value_serializer=JSONSerializer(schema=MY_SCHEMA))
```

> ### Avro

Reviewer: ```suggestion
## Avro
```

> The avro serializer and deserializer need to be pass explicitly.

Reviewer: ```suggestion
The Avro serializer and deserializer need to be passed explicitly.  

In the current version, the schema must be provided manually.
```

input_topic = app.topic('input', value_deserializer=JSONDeserializer(schema=MY_SCHEMA))
output_topic = app.topic('output', value_serializer=JSONSerializer(schema=MY_SCHEMA))
```

## Avro

> 

Reviewer: ```suggestion
Apache Avro is a row-based binary serialization format data. Avro stores the schema in JSON format alongside the data, enabling efficient processing and schema evolution.

You can learn more the Apache Avro format [here](https://avro.apache.org/docs/).
```

The Avro serializer and deserializer need to be passed explicitly.  

In the current version, the schema must be provided manually.

> > ***WARNING***: The avro serializer and deserializer requires the fastavro library.

Reviewer: ```suggestion
> ***WARNING***: The avro serializer and deserializer require the `fastavro` library.
```

> > ***WARNING***: The avro serializer and deserializer require the `fastavro` library.

Reviewer: ```suggestion
> ***WARNING***: Avro serializer and deserializer require the `fastavro` library.
```
